Emby.Dlna/DlnaManager.cs

```diff
@@ -216,11 +216,11 @@ private bool IsMatch(DeviceIdentification deviceInfo, DeviceIdentification profi
             return true;
         }
 
-        private bool IsRegexMatch(string input, string pattern)
+        private bool IsPropertyMatch(string input, string pattern)
         {
             try
             {
-                return Regex.IsMatch(input, pattern);
+                return Regex.IsMatch(input, pattern, RegexOptions.IgnoreCase | RegexOptions.CultureInvariant) || input.Contains(pattern, StringComparison.OrdinalIgnoreCase);
```

Comment:
```suggestion
                return input.Contains(pattern, StringComparison.OrdinalIgnoreCase) || Regex.IsMatch(input, pattern, RegexOptions.IgnoreCase | RegexOptions.CultureInvariant);
```
Put the cheaper operation first so it can short circuit when it return true